devel/uboot-tools/Portfile

```diff
@@ -17,20 +21,22 @@ long_description    U-Boot is a boot loader for Embedded boards based on \
                     port compiles some binary tools that can be used to \
                     deploy u-boot on embedded devices.
 
-checksums           rmd160  124de3a5355915ccb2291b4a01f21e7710290521 \
-                    sha256  616b446e15d1cd1ab6461ebb61ac6655a2b13e902fe0601f36c4affb3949d416 \
-                    size    19630685
+checksums           rmd160  25357e70695de9aac1cee2a752cb4fd82def66ed \
+                    sha256  0c022ca6796aa8c0689faae8b515eb62ac84519c31de3153257a9ee0f446618f \
+                    size    20176030
 
 depends_build       port:gmake
 
 depends_lib         path:lib/libssl.dylib:openssl
 
+compiler.c_standard 2011
+
 configure.cmd       gmake defconfig
 configure.pre_args
 
 build.target        tools
 build.cmd           gmake
-build.args-append   KBUILD_HOSTCFLAGS=-I${prefix}/include
+build.args-append   KBUILD_HOSTCFLAGS="'-I${prefix}/include'"
```

Comment:
Why add these quotes?

Reply:
I don't actually remember, but I guess there was a reason? 😅

Reply:
Uhm, I can't seem to find anything about that... could you try without the quotes?